bedrock/products/views.py

```diff
@@ -532,3 +542,39 @@ def mozsocial_waitlist_page(request):
     ctx = {"action": settings.BASKET_SUBSCRIBE_URL, "newsletter_form": newsletter_form, "product": "mozilla-social-waitlist"}
 
     return l10n_utils.render(request, template_name, ctx, ftl_files=ftl_files)
+
+
+@require_safe
+def vpn_resource_center_redirect(request, slug):
+    VPNRC_SLUGS = {
+        "what-is-an-ip-address": {
+            "slug": "what-is-an-ip-address",
+            "old_template": "products/vpn/more/ip-address.html",
+            "ftl_files": ["products/vpn/more/ip-address", "products/vpn/shared"],
+        },
+        "vpn-or-proxy": {
+            "slug": "the-difference-between-a-vpn-and-a-web-proxy",
+            "old_template": "products/vpn/more/vpn-or-proxy.html",
+            "ftl_files": ["products/vpn/more/vpn-or-proxy", "products/vpn/shared"],
+        },
+        "what-is-a-vpn": {
+            "slug": "what-is-a-vpn",
+            "old_template": "products/vpn/more/what-is-a-vpn.html",
+            "ftl_files": ["products/vpn/more/what-is-a-vpn", "products/vpn/shared"],
+        },
+        "when-to-use-a-vpn": {
+            "slug": "5-reasons-you-should-use-a-vpn",
+            "old_template": "products/vpn/more/when-to-use.html",
+            "ftl_files": ["products/vpn/more/when-to-use-a-vpn", "products/vpn/shared"],
+        },
+    }
+    locale = l10n_utils.get_locale(request)
+    curr_page = VPNRC_SLUGS[slug]
+    rc_slug = curr_page["slug"]
+    redirect_link = f"{locale}/products/vpn/resource-center/{rc_slug}/"
+    if active_locale_available(rc_slug, locale):
```

Comment:
I think we need to think through what the desired behaviour here is so we can check that the logic is matching up.

I'll call the new templates rc and the old more

When a /resource-centre url is requested the user should have the rc article displayed in the requested language. 
- If the rc article is not available in their requested language bedrock should display the more article is available in their language. 
- If neither is available in their language bedrock should forward to the English rc article.

When a /more url is requested the user should be forwarded to the /resource-centre url. The above logic then applies.

Is that right?

Reply:
Yes, that is right. I'll add a comment to the view to make that clear

Reply:
I think the focus of this PR was to just handle the redirects for the /more/ urls - but I can check with adria to make sure.

Reply:
Re-reading the "significant changes" section of the PR I can see that what I outlined is not the behaviour you outlined and it is desired to display something at the /more/ URL in some cases. I'm sorry I made a bad logical leap about the desired behaviour.